Automattic/jetpack
# Fix lazy images issue with Gazette theme

Fixes blank featured images in top area of Gazette theme when Lazy Images is switched on.

https://wordpress.org/support/topic/no-header-images-with-jetpack-lazy-images/#post-9849480

The issue is that Gazette rendered out the original image, then in javascript they copy the image `src` attribute and put it in the `style` attribute of the parent tag as `background-image: url( 'the-original-image' )`, then hide the original image tag - which in the case of lazy images means they are essentially copying the placeholder into the parent where we can't modify it.

This PR is my attempt at the simplest, narrowest solution targeting this theme. Open to better suggestions!

#### Testing instructions:

To see the issue:

* Activate Gazette theme
* Add some posts with featured images and "featured' tag
* Go to your main blog index
* You should see blocks above the header bar with the featured posts in them
* Activate lazy images
* Reload blog page - post featured images should now be missing (black square)

To test this patch:

* Do the above - images should now be present.


#### Proposed changelog entry for your changes:
